src/sentry/static/sentry/app/plugins/components/pluginIcon.tsx

```diff
@@ -73,10 +73,15 @@ export const ICON_PATHS = {
   youtrack,
 };
 
+type IntegrationIconProps = {
+  image: string;
+  size: number;
+};
+
 const IntegrationIcon = styled('div')`
```

Comment:
Can you just give it the props template parameter?

```jsx
const IntegrationIcon = styled('div')<IntegrationIconProps>`
```

Then we can avoid needing to type each template literal expansion function

Reply:
@EvanPurkhiser good suggestion, fixed